src/main/java/games/strategy/engine/lobby/server/userDB/DbUser.java

```diff
@@ -0,0 +1,41 @@
+package games.strategy.engine.lobby.server.userDB;
+
+import java.io.Serializable;
+import java.util.Date;
+
+public class DbUser implements Serializable {
```

Comment:
For my own benefit, are instances of this class actually serialized/deserialized (via capital-S Serialization)?  I could only find one place (in `DBUserController#getUser()` where such instances are constructed directly from a `ResultSet`.  If they are never serialized, can we get rid of `Serializable` since you are refactoring the entire class at this time?  I've found that it makes future refactorings much easier (one doesn't have to research what I just did :smiley:).